lib/core/src/Cardano/Wallet/DB/Sqlite/CheckpointsOld.hs

```diff
@@ -470,17 +447,34 @@ selectSeqStatePendingIxs wid =
   where
     fromRes = fmap (W.Index . seqStatePendingIxIndex . entityVal)
 
-selectSeqAddressList
-    :: forall c. Typeable c
-    => W.WalletId -> W.SlotNo -> SqlPersistT IO (SeqAddressList c)
-selectSeqAddressList wid sl = do
-    SeqAddressList . map (toPair . entityVal) <$> selectList
+insertSeqAddressMap
+    :: forall n c key. (PaymentAddress n key, Typeable c)
+    =>  W.WalletId -> W.SlotNo -> SeqAddressMap c key -> SqlPersistT IO ()
+insertSeqAddressMap wid sl (SeqAddressMap pool) = void $
+    dbChunked insertMany_
+        [ SeqStateAddress wid sl (liftPaymentAddress @n addr)
```

Comment:
correct here for payment key hash credential provided we have ONE staking credential (also key hash).

Reply:
yes, I just want to underline that we have fingerprint properly handled for enterprise and base addresses (with key hash credentials), and only for the situation of one staking credential;-)